README.md

```diff
@@ -8,26 +8,31 @@ Topics:
 
 ## Instructions
 
-* Run `npm i` inside the root directory of this project. Run `npm start` to start the server.
-* In a separate terminal window, run `create-react-app friends` to create your starter application.
-* The provided server returns a list of friends when making a `GET` request to `http://localhost:5000/friends`.
-* Inside your React application create a component to display the list of friends coming from the server.
-* Add a form to gather information about a new friend and a button to save the friend by making a `POST` request to the same endpoint listed above.
-* Each `friend` should have the following properties:
+1. Run `npm i` inside the root directory of this project.
+2. Run `npm start` to start the server.
+3. The provided server returns a list of friends when a `GET` request is made to [`http://localhost:5000/friends`](http://localhost:5000/friends).
+4. In a separate terminal window, run `create-react-app friends` to create your starter React application.
+5. Inside your React application, create a component to display the list of friends coming from the server.
+6. Add a form to gather information about a new friend.
+7. Add a button to save the new friend by making a `POST` request to the same endpoint listed above.
+7. Each `friend` should have the following properties:
 
-```
+```js
```

Comment:
Have you verified that this works?  It doesn't need to be `javascript`?

Reply:
Ya, GitHub Flavored Markdown supports grammar coloring. I use js, json, c, bash, html, etc. after the opening set of triple backticks. Per:
https://help.github.com/articles/creating-and-highlighting-code-blocks/

...In this case tho, it doesn't do much except make the colons red (that sounds weird).

Reply:
Ok awesome.  I've written it out as `javascript` instead of `js` so just wanted to double check.